content/en/docs/concepts/security/windows-security.md

```diff
@@ -0,0 +1,33 @@
+---
+reviewers:
+- jayunit100
+- jsturtevant
+- marosset
+- perithompson
+title:    Security For Windows Nodes
+content_type: concept
+weight: 75
+---
+
+<!-- overview -->
+
+This page describes security considerations and best practices specific to the Windows operating system.
+
+<!-- body -->
+
+On Windows, data from Secrets are written out in clear text onto the node's local
+storage (as compared to using tmpfs / in-memory filesystems on Linux). As a cluster
+operator, you should take both of the following additional measures:
+
+1. Use file ACLs to secure the Secrets' file location.
+1. Apply volume-level encryption using [BitLocker](https://docs.microsoft.com/en-us/windows/security/information-protection/bitlocker/bitlocker-how-to-deploy-on-windows-server).
+
```

Comment:
```suggestion

## Container isolation by user identity

```
?

Reply:
Hmm, specifying the username doesn't really impact isolation, let me think about what to use for the section header.

Reply:
I re-named this section to `Container Users` and added more details.

PTAL

Reply:
Sounds good